pkg/plugin/driver/job/job.go

```diff
@@ -63,7 +63,10 @@ func NewPlugin(dfn plugin.Definition, namespace, sonobuoyImage, imagePullPolicy
 // a Job only launches one pod, only one result type is expected.
 func (p *Plugin) ExpectedResults(nodes []v1.Node) []plugin.ExpectedResult {
```

Comment:
Hi @johnSchnake,

Consider rewriting the function signature as func `func (p Plugin) ExpectedResults(nodes []v1.Node) []plugin.ExpectedResult` since there is no reason for the function to be able to write any values to the address of `p`. A read on a copy will suffice and offers better protection.